tools/cuttlefish-host-image-installer/update_github_releases.sh

```diff
@@ -20,6 +20,19 @@ echo "$changes" > changelog
 
 STABLE_VERSION="v$version"
 
+echo "step 2: move stable release to new commit id"
+commit_id=$(git  rev-list -n 1 $STABLE_VERSION )
+stable_commit_id=$(git  rev-list -n 1 stable )
+if [[ "$commit_id" == "$stable_commit_id" ]]; then
+  echo "same commit, no need to change code base"
+else
+  git tag -d stable
+  git push -d origin stable
+  git push origin :refs/tags/stable
+  git tag stable $commit_id
```

Comment:
`git tag -f` and `git push -f` allow overriding tags, then it doesn't need two steps to delete and recreate.

Reply:
Done.